app/assets/stylesheets/helpers/_mixins.scss

```diff
@@ -9,11 +9,11 @@
     width: 960px;
   }
 
-  @include media(tablet) {
+  @include govuk-media-query($from: tablet) {
     margin: 0 30px;
   }
 
-  @include media($min-width: (960px + 30px * 2)) {
+  @media (min-width: (960px + 30px * 2)) {
```

Comment:
we could you $govuk-page-width and $govuk-gutter

Reply:
Thanks. My initial reaction to this line was I didn't want to touch it with a barge pole! But I'll update it.

Reply:
i'd try `@include govuk-media-query(($govuk-page-width + $govuk-gutter * 2));` and hope

Reply:
Could we just use `$from: desktop` ?

Reply:
nein, because
// As soon as the viewport is greater than the width of the page plus the
// gutters, just centre the content instead of adding gutters.

Reply:
Updated, thanks. I'll make sure this gets tested thoroughly with some apps.